tests/test_ec2/test_amis.py

```diff
@@ -878,6 +878,30 @@ def test_ami_snapshots_have_correct_owner():
             assert owner_id == snapshot["OwnerId"]
 
 
+@mock_ec2
+def test_ami_filter_by_empty_tag():
+    ec2 = boto3.resource("ec2", region_name="us-west-1")
+    client = boto3.client("ec2", region_name="us-west-1")
+
+    instance = ec2.create_instances(ImageId=EXAMPLE_AMI_ID, MinCount=1, MaxCount=1)[0]
+    image_id = client.create_image(
+        InstanceId=instance.instance_id, Name="test-image", Description="test ami",
+    )["ImageId"]
```

Comment:
Can you create a few images, and verify that we're only returning the one with the tags applied?
The fix looks alright to me, just want to make sure that we're explicitly testing the error case that was supplied:

> So instead of filtering only those images that have a RELEASE tag with an empty value, all images are returned whether they have this filter or not.